controllers/utils/base_reconciler.go

```diff
@@ -262,10 +262,11 @@ func (r *ReconciliationRunner) DeleteOwnedObjects(gvks ...schema.GroupVersionKin
 
 			// Filter objects not actually owned by reconciliation subject via owner reference UID.
 			for _, pOwned := range potentiallyOnwedObjs.Items {
+				_pOwned := pOwned
```

Comment:
what's the point of this variable?

Reply:
To get rid of a lint warning. 

If we use address of a loop variable, lint will complain since that address does not change in each loop. The value pointed by that address DOES change, so the code will run correctly without _pOwned.

https://github.com/golang/go/issues/20725